definitions/variable/afolu/forestry.yaml

```diff
@@ -1,19 +1,82 @@
-- Forestry Demand|Roundwood:
+# Forestry Demand Roundwood
+- Forestry Demand Roundwood:
     description: Total roundwood demand
     unit: million m3/yr
-- Forestry Demand|Roundwood|Industrial Roundwood:
-    description: Industrial roundwood demand
+    tier: 1
+- Forestry Demand Roundwood|{Raw Products}:
+    description: "{Raw Products} demand"
     unit: million m3/yr
-- Forestry Demand|Roundwood|Wood Fuel:
-    description: Wood fuel demand
+    tier: 2
+- Forestry Demand Roundwood|Industrial Roundwood|Material use:
+    description: Industrial roundwood demand - used for material
```

Comment:
Please make the description of bit more helpful.

Reply:
Done: Industrial roundwood demand - used as material for wood products